proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -35,3 +35,26 @@ message AuthorizationGrant {
   google.protobuf.Any       authorization = 1 [(cosmos_proto.accepts_interface) = "Authorization"];
   google.protobuf.Timestamp expiration    = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
 }
+
+// DelegateAuthorization allows the grantee to delegate tokens to provided set of validators
+// on behalf of the granter's account.
+message DelegateAuthorization {
```

Comment:
These should live in `x/staking`. Now that I think of it `SendAuthorization` should live in `x/bank`. Can we move it? It makes sense to have authorizations live in the modules they belong to and only `GenericAuthorization` should live in `x/authz`

Reply:
This is creating import cycles.

Reply:
How come?

Reply:
This is resolved.